src/lib/types/minions.ts

```diff
@@ -92,6 +92,13 @@ export interface MiningActivityTaskOptions extends ActivityTaskOptions {
 	powermine: boolean;
 }
 
+export interface MotherlodeMiningActivityTaskOptions extends ActivityTaskOptions {
```

Comment:
What "oreID" does MLM have? its not an ore.

Reply:
The "oreID" of Gem Rock is mapped to "uncut opal" so i didnt think we was using oreID in a strict manner. I used the ID of pay-dirt though as I assumed it was the same concept.

Reply:
There's no reason at all to do this. Even if there was, its not an acceptable solution. MLM is not an ore, make it its' own thing.